### PR TITLE
length add for custom capture filter

### PR DESCRIPTION
**Pull request by shramroy**

### CODE REVIEW

File: capture/capture.yaml [hunk 1]
```diff
@@ -83,16 +83,23 @@ components:
             The byte offset to filter on
           type: integer
           default: 0
+        byte_length:
```
Comment (ashutshkumr) [inline capture/capture.yaml:86]: change this to bit_length

**Review by ashutshkumr — CHANGES_REQUESTED** [review]

File: capture/capture.yaml (hunk 1 above)
Comment (shramroy) [reply capture/capture.yaml]: updated accordingly

File: capture/capture.yaml [hunk 2]
```diff
@@ -83,16 +83,23 @@ components:
...
+          type: integer
+          default: 8
     
     Capture.Field:
       type: object
       properties:
         value:
           type: string
           format: hex
+          default: 0
```
Comment (ashutshkumr) [inline capture/capture.yaml:98]: should the defaults be "0" or 0 ?

File: capture/capture.yaml [hunk 3]
```diff
@@ -83,16 +83,23 @@ components:
             The byte offset to filter on
           type: integer
           default: 0
+        bit_length:
+          description: >-
+            The bit length to filter on
```
Comment (ashutshkumr) [inline capture/capture.yaml:88]: The bit length of field to filter on

Comment (ashutshkumr) [reply capture/capture.yaml:88]: change the description for byte offset as well. "The byte offset of field to filter on"

File: capture/capture.yaml (hunk 2 above)
Comment (shramroy) [reply capture/capture.yaml:98]: In the other fields like mac and ip default was mentioned without quotation. That's why I didn't use quotation.

File: capture/capture.yaml (hunk 3 above)
Comment (shramroy) [reply capture/capture.yaml]: updated

File: capture/capture.yaml (hunk 2 above)
Comment (shramroy) [reply capture/capture.yaml]: updated

**Review by ashutshkumr — APPROVED** [review]

**Pull request merged**